v2/util/docker-cleanup.sh

```diff
@@ -6,9 +6,12 @@ if [[ $(/usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}
 then
   echo "Starting docker cleanup with a max of: $MAX_DOCKER"
   /usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}' | grep week | awk '{print $1}' | head -n $MAX_DOCKER | xargs /usr/bin/docker rm -v
-  /usr/bin/docker images -a | grep none | awk '{print $3}' | xargs /usr/bin/docker rmi
-  echo "cleanup complete"
+  echo "container cleanup complete"
 else
-  echo "nothing to cleanup"
+  echo "no containers to cleanup"
 fi
+
+/usr/bin/docker images -a  | awk '{print $3}' | xargs /usr/bin/docker rmi
```

Comment:
this PR is fine, but probably won't help much. you can't remove an image if there's still a container that relies on it, which is why the `docker rmi` command is wrapped in the `if` block.

see:
![image](https://cloud.githubusercontent.com/assets/2601345/10124554/adf0b15c-6525-11e5-9d16-85a4001fe239.png)